schemas/tls.yml

```diff
@@ -0,0 +1,174 @@
+---
+- name: tls
+  title: TLS
+  group: 2
+  short: Fields describing a TLS connection.
+  description: >
+    Fields related to TLS activity.
```

Comment:
I prefer the wording of the "short" description, using the word "connection" rather than "activity", so let's align the main description to use this word as well.

Note that "short" is used in places like [the listing of all field sets](http://ecs_606.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-field-reference.html), so the current curt sentence is perfect there. However "description" is meant to populate [the paragraph(s) used to introduce the field set](http://ecs_606.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-tls.html#ecs-tls) at length.

Feel free to flesh out "description" some more, if needed. If you feel nothing else is needed at this time, then perhaps we can simply align on usage of the word "connection" and call it done. I guess Packetbeat's "main" [description for the field set](https://www.elastic.co/guide/en/beats/packetbeat/current/exported-fields-tls.html) is just as short :-)